libs/react-components/src/lib/components/Button/styles.module.css

```diff
@@ -43,3 +43,12 @@
     border-color: var(--td-web-primary-navy);
   }
 }
+
+.tdButtonText{
+  background: transparent;
+  color: var(--td-web-primary-navy);
+  border: none;
+  &:hover {
+    background: rgba(0, 35, 60, 0.15);
```

Comment:
@ObedVega Can we replace this hard coded value with css variables. I think you can use `--td-web-primary-navy` and set the opacity value to `0.15`

Reply:
.tdButtonText{
  background: transparent;
  color: var(--td-web-primary-navy);
  border: none;
  &:hover {
    background: var(--td-web-primary-navy-hover); 
  }
}